### ISSUE TITLE
p_load has no way of controlling dependencies argument in install.packages

### ISSUE THREAD

**Issue by trinker** [trinker/pacman#43]
I think it would be helpful to have `p_load` be able to specify `dependencies` from the `install.packages` function.  I'd say the defualt should be `TRUE` where as I think it's `NA` in the `install.packages` function but this is certainly open for discussion.

Right now the question is should `p_load` allow for installing dependencies?  We don't want to bloat but this seems like something people would definitely want to supply.  Normally we could just use `...` to pass extra arguments but we've already used `...` in the function.

Thoughts?

**Comment by Nosferican** [comment]
I agree that an option to specify 'dependencies = TRUE' should be incorporated in the p_load arguments.
